tests/basic/storage_pool_test.go

```diff
@@ -9761,3 +9761,110 @@ var _ = Describe("{AddDriveMetadataPool}", func() {
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "Get Px State after pool delete"
+	It(stepLog, func() {
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("rebalanceshutdown-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		// Get Node Details of the Pool with IO
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+		log.InfoD("Pool with UUID [%v] present in Node [%v]", poolUUID, nodeDetail.Name)
+
+		if IsLocalCluster(*nodeDetail) != true {
+			log.FailOnError(fmt.Errorf("This test will only support onprem vms"), "is this onprem?")
+		}
+		// Get Total Pools present on the Node present
+		poolDetails, err := GetPoolsDetailsOnNode(*nodeDetail)
+		log.FailOnError(err, "Failed to get Pool Details from Node [%v]", nodeDetail.Name)
+		log.InfoD("List of Pools present in the node [%v]", poolDetails)
+		poolToBeResized, err := GetStoragePoolByUUID(poolDetails[0].Uuid)
+		drvSize, err := getPoolDiskSize(poolToBeResized)
+		log.FailOnError(err, "error getting drive size for pool [%s]", poolToBeResized)
+		expectedSize := (poolToBeResized.TotalSize / units.GiB) + drvSize
+		err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, poolToBeResized.Uuid, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, true)
+		log.FailOnError(err, "error getting drive size for pool [%s]", poolToBeResized)
+		expandedPool, err := GetStoragePoolByUUID(poolToBeResized.Uuid)
+		log.FailOnError(err, "error getting pool by using id %s", poolToBeResized.Uuid)
+		dash.VerifyFatal(expandedPool == nil, false, "expanded pool value is nil")
+		poolStatus, err := getPoolLastOperation(expandedPool.Uuid)
+		log.FailOnError(err, "failed to get last operation on pool %s", expandedPool.Uuid)
+		if poolStatus.Msg != "" {
+			log.Infof("Pool Resize Status: %v, Message : %s", poolStatus.Status, poolStatus.Msg)
+			if poolStatus.Status == api.SdkStoragePool_OPERATION_IN_PROGRESS &&
+				(strings.Contains(poolStatus.Msg, "Storage rebalance is running") || strings.Contains(poolStatus.Msg, "Rebalance in progress")) {
+				errstring := true
+				dash.VerifyFatal(errstring == true, true, "poolresize failed")
+			}
+			var connect node.ConnectionOpts
+			connect.Timeout = 60
+			connect.TimeBeforeRetry = 10
+			err := Inst().N.ShutdownNode(*nodeDetail, node.ShutdownNodeOpts{
+				Force:          true,
+				ConnectionOpts: connect,
+			})
+			time.Sleep(300 * time.Second)
+			log.FailOnError(err, "failed to shutdown the node with err %s", err)
```

Comment:
do error validation first then sleep, also add a log for why you have added sleep